.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           name: run mypy
           command: |
               mypy --version
-              mypy --exclude=^docs/|setup.py$
+              mypy --exclude="^docs/|setup.py$"
```

Comment:
Should already be excluded as they aren't implicitly included in mypy.ini. If you want to explicitly exclude it, do it there.